portfolio/src/main/webapp/me.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="node_modules/bulma/css/bulma.min.css" />
```

Comment:
These tags don't need to have the trailing slash, and according to the Google HTML style guide they shouldn't have them.